src/Files.App.Storage/NativeStorage/NativeFolder.cs

```diff
@@ -0,0 +1,204 @@
+﻿using Files.Sdk.Storage;
+using Files.Sdk.Storage.Enums;
+using Files.Sdk.Storage.Extensions;
+using Files.Sdk.Storage.LocatableStorage;
+using Files.Sdk.Storage.ModifiableStorage;
+using Files.Sdk.Storage.MutableStorage;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Files.App.Storage.NativeStorage
+{
+    /// <inheritdoc cref="IFolder"/>
+    public sealed class NativeFolder : NativeStorable, ILocatableFolder, IModifiableFolder, IMutableFolder
+    {
+        public NativeFolder(string path)
+            : base(path)
+        {
+        }
+
+        /// <inheritdoc/>
+        public Task<IFile> GetFileAsync(string fileName, CancellationToken cancellationToken = default)
+        {
+            var path = System.IO.Path.Combine(Path, fileName);
```

Comment:
The `System.IO` is not necessary since it's already imported with the `using`.

Reply:
`NativeStorage` defines a property named `Path` so this is needed